src/ui/public/chrome/directives/app_switcher/app_switcher.html

```diff
@@ -9,8 +9,8 @@
       ng-href="{{ link.active ? link.url : (link.lastSubUrl || link.url) }}"
       data-test-subj="appLink">
 
-      <div ng-if="link.icon" ng-style="{ 'background-image': 'url(../' + link.icon + ')' }" class="app-icon"></div>
-      <div ng-if="!link.icon" class="app-icon app-icon-missing">{{ link.title[0] }}</div>
+      <div ng-if="link.icon" ng-style="{ 'mask-image': 'url(../' + link.icon + ')' }" class="app-icon"></div>
```

Comment:
I have no strong opinion on this. From a user standpoint it works the same. @simianhacker any reason you went this direction?